docs/source/plugins/03_plugin_structure.rst

```diff
@@ -4,48 +4,41 @@ Plugins Structure
 =================
 
 As mentioned in :ref:`quick-start-section`, the |sdk| package has some utilities to help in the process to create
-a new plugin project and to create a plugin file.
+a plugin file.
 
-At any moment, it is possible to invoke the ``help`` command to list all commands available.
+First, for the creation of the plugin you can use the CLI automatic generator
 
 .. code-block:: bash
 
-   >>> alfasim-sdk --help
+   >>> alfasim-sdk new --help
 
+.. _alfasim_sdk_cli_new_section:
 
-You can also learn about the available options of each command invoking the :guilabel:`--help` option for each command:
-
-.. code-block:: bash
-
-   >>> alfasim-sdk template --help
-   >>> alfasim-sdk package --help
-   >>> alfasim-sdk compile --help
-   >>> alfasim-sdk update --help
-
-.. _alfasim_sdk_cli_template_section:
-
-.. click:: alfasim_sdk._internal.cli:template
-    :prog: alfasim-sdk template
+.. click:: alfasim_sdk._internal.cli:new
+    :prog: alfasim-sdk new
     :show-nested:
 
+Pyinvoke tasks
+--------------
 
-.. _alfasim_sdk_cli_package_section:
+Besides the creation of the plugin struct, all other activities that you might want to perform, you will use ``pyinvoke`` tasks.
 
-.. click:: alfasim_sdk._internal.cli:package
-    :prog: alfasim-sdk package
-    :show-nested:
+Below there is detailed list on what they are meant to do. However, in general, the tasks that you will use the most are ``compile``,
+``package`` and ``update``.
 
+.. invoke::
+   :module: alfasim_sdk.default_tasks
+   :prog: invoke
 
-.. click:: alfasim_sdk._internal.cli:package_only
-    :prog: alfasim-sdk package_only
-    :show-nested:
+You can check their implementations `here <https://github.com/ESSS/alfasim-sdk/blob/master/src/alfasim_sdk/default_tasks.py>`_
+and you can also overwrite them by just defining a function with the same name of the default task with the ``@task`` decorator.
```

Comment:
Okay, I will leave as it is.

Reply:
"read the docs" will build the docs for your PR, just add a PR comment on the same line with the link from that.

Reply:
I am not sure if I understand what you are suggesting. I was talking that the link to <https://github.com/ESSS/alfasim-sdk/blob/master/src/alfasim_sdk/default_tasks.py> is broken because I am linking a file that still doesnt exist on github at the master branch.

Reply:
This is a minor thing, I will resolve the conversation, because once merged the link will be valid.